Main.java

```diff
@@ -0,0 +1,10 @@
+class Main
```

Comment:
You can be specific here, if there is no reason to make it **default**, then make the Main class **public**.